src/atoms/LinkBarDropdown.js

```diff
@@ -116,6 +116,7 @@ Dropdown.propTypes = {
 		propTypes.node,
 		propTypes.arrayOf(propTypes.node),
 	]).isRequired,
+	/** Expands the menu by default. */
```

Comment:
Fill out comments for rest of the props?

Reply:
Done, thanks